apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx

```diff
@@ -88,6 +88,11 @@ const CalendarPaneToolbar = ({
         setOpenSchedules((prev) => !prev);
     };
 
+    const handleClose = (event: React.ChangeEvent<unknown>) => {
```

Comment:
How is the intended effect of this different than `handleMenuClose` and why are their implementations so different, desipite almost identical names? Might be worth adding comments to this component to make it clearer, since it's hard to tell why this new function is necessary.

Reply:
Definitely was too vague in my first commit! 

I've gone and renamed handleClose to handleScheduleClose to be more specific. There are now two pairs of handleClick & handleClose, with one set for the Menu (which is the mobile menu that has the download, screenshot, and custom event adder) and another set for the Schedule selector element which is what I'm focusing on in this PR.

If we want to be more verbose and clear about what's what, handleMenuClose/Click could be renamed to handleMobileMenuClose/Click. handleScheduleClose/Click could become handleScheduleSelectClose/Click as well!

LMK what you think and I'll get right on it!